### DISCUSSION TITLE
setting default value for boolean type radio input

### DISCUSSION THREAD

**Discussion by sxshateri** [filamentphp/filament#1893] [Help]
isn't it possible to set a default value for a boolean type radion input?
I'm trying the following but it doesn't seem to work:
```
Radio::make('usi')->label('Do you have a USI ? (Unique Student Identifier)')->boolean()->inline()->default(false)->required(),
```
am i doing it wrong or it's just not possible?

**Comment by intrepidws** [comment]
I just tried the following code on the latest version of Filament and it worked fine, defaulting to No:

```php
Forms\Components\Radio::make('testing')
    ->boolean()
    ->default(false),
```

**Comment by danharrin** [comment]
Make sure you're calling `$this->form->fill()` if you're using the form builder on its own.
